README.md

```diff
@@ -171,344 +54,31 @@ The following guide assumes you have some sort of ES2015 build set up using babe
 You can also start with [preact-boilerplate] or a [CodePen Template](http://codepen.io/developit/pen/pgaROe?editors=0010).
```

Comment:
A link to `codesandbox` instead of `codepen` would be great here.

Codepen is too old :P